### DISCUSSION TITLE
Use Labels for Issue/PR Categorization

### DISCUSSION THREAD

**Discussion by HeitorAugustoLN** [nix-community/plasma-manager#305] [Ideas]
I’m considering introducing labels to help us categorize our issues and pull requests more effectively. I’ve drafted a set of labels for this purpose and would love to get your feedback on them.

What are your thoughts? @magnouvean @toast003

**Comment by magnouvean** [comment]
They look fine to me. The only thing I though was a little weird was the `4` labels, which is of type "has", which doesn't fit that well with the label names. Would "requires" maybe be better?

**Reply by magnouvean** [reply]
Tbh looking back at it "requires" doesn't really fit either. "request" or something similar could work

**Reply by HeitorAugustoLN** [reply]
It was supposed to be like the `has: package (new)` from nixpkgs for PRs. But I don't think it really fits tbh. But I think we should have something similar, but I don't have other ideas rn

**Reply by HeitorAugustoLN** [reply]
But I think we can add something like `needs: plasma module (new)` for feature request issues

**Reply by HeitorAugustoLN** [reply]
That way we don't need to change the `4` labels

**Reply by magnouvean** [reply]
Yeah. Ultimately it's not that important and we can just change them whenever we want, but maybe "needs" fits better

**Reply by HeitorAugustoLN** [reply]
I've added some more labels. I think it is good to go now.

**Reply by HeitorAugustoLN** [reply]
If you think it is good, we can start using them.

**Comment by toast003** [comment]
Labels look good to me
